<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartSetting2</class>
 <widget class="QDialog" name="ChartSetting2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图表设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局设置：强制白底黑字 */
QDialog, QWidget {
        background-color: white;
        color: black;
        font-family: &quot;Microsoft YaHei&quot;;
}
QLabel {
        color: black;
}
QLineEdit, QDoubleSpinBox {
        color: black;
        background-color: white;
        border: 1px solid #555;
        border-radius: 2px;
        padding: 2px;
}
/* TabWidget 样式 */
QTabWidget::pane {
        border: 1px solid #999;
}
QTabBar::tab {
        background: #f0f0f0;
        color: black;
        padding: 8px 15px;
        border: 1px solid #bbb;
        border-bottom-color: #999;
}
QTabBar::tab:selected {
        background: white;
        border-bottom-color: white;
}
/* GroupBox 样式 */
QGroupBox {
        border: 1px solid #999;
        margin-top: 10px;
        padding-top: 10px;
}
QGroupBox::title {
        subcontrol-origin: margin;
        left: 10px;
        padding: 0 3px;
        color: black;
        font-weight: bold;
}
QPushButton {
        color: black;
        background-color: #f0f0f0;
        border: 1px solid #555;
        padding: 5px 15px;
        border-radius: 3px;
}

/* 复选框美化：解决选中看不清的问题 */
QCheckBox {
        color: black;
        spacing: 5px;
}
QCheckBox::indicator {
        width: 16px;
        height: 16px;
        border: 1px solid #555;
        border-radius: 3px;
        background: white;
}
QCheckBox::indicator:checked {
        background-color: #0078d7;
        border: 1px solid #0078d7;
    /* 使用纯蓝色背景表示选中，无需图片资源 */
}

/* 下拉框美化 */
QComboBox {
    color: black;
    background-color: white;
    border: 1px solid #555;
    border-radius: 2px;
    padding: 2px;
}
QComboBox QAbstractItemView {
    color: black;
    background-color: white;
    selection-background-color: #0078d7;
    selection-color: white;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>通用设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>基本信息</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>图表标题:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineChartTitle"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>X轴设置 (时间)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>标签:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineXLabel"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>范围:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinXMin">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="spinXMax">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="checkXVisible">
              <property name="text">
               <string>显示轴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkXTickLabels">
              <property name="text">
               <string>显示刻度值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkXGrid">
              <property name="text">
               <string>网格</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkXSubGrid">
              <property name="text">
               <string>子网格</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPressure">
      <attribute name="title">
       <string>压力图设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Y轴设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>标签:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="linePLabel"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>范围:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinPMin">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="spinPMax">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkPVisible">
              <property name="text">
               <string>显示轴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkPTickLabels">
              <property name="text">
               <string>显示刻度值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkPGrid">
              <property name="text">
               <string>网格</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkPSubGrid">
              <property name="text">
               <string>子网格</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>259</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProduction">
      <attribute name="title">
       <string>产量图设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Y轴设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>标签:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineQLabel"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>范围:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinQMin">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="spinQMax">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkQVisible">
              <property name="text">
               <string>显示轴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkQTickLabels">
              <property name="text">
               <string>显示刻度值</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkQGrid">
              <property name="text">
               <string>网格</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkQSubGrid">
              <property name="text">
               <string>子网格</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>259</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChartSetting2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartSetting2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
